packages/create-amplify/src/process_arguments.ts

```diff
@@ -0,0 +1,13 @@
+import yargs from 'yargs';
+import * as process from 'process';
+
+export const processArguments = await yargs(process.argv.slice(2)).options({
+  'cdk-version': {
```

Comment:
I had similar doubt but wanted to see thoughts.

Two options to proceed:
1. make it undocumented/hidden parameter (i.e. remove from `npm create amplify -- --help`)
2. in the test - alter the package.json, re-install and run validation steps again (this can be in a form of added test)


I'd pick 1. due to simplicity. Let me know which one you prefer or whether you have other ideas.

Reply:
For # 2 could the test just run
```shell
npm create amplify
npm install aws-cdk-lib@<baseline version> aws-cdk@<baseline version>
```
Then proceed with the test as usual?